doc/release-notes.md

```diff
@@ -4,3 +4,9 @@ release-notes at release time)
 Notable changes
 ===============
 
+UTXO memory accounting
+----------------------
+
+The default -dbcache has also been changed in this release to 450MiB. Users can set -dbcache to a higher value (e.g. to keep the UTXO more fully cached in memory). Users on low-memory systems (such as systems with 1GB or less) should consider specifying a lower value for this parameter.
```

Comment:
Non-blocking: `has been changed` (non-blocking because I expect other changes will be added that can go before this, to make it grammatically correct).

Reply:
Done